src/Components/test/testassets/ComponentsApp.Server/Pages/Index.cshtml

```diff
@@ -1,3 +1,8 @@
+@page "{*clientroutes}"
+@using ComponentsApp.App
+@model ComponentsApp.Server.Pages.IndexModel
+@{
+}
```

Comment:
Presumably the extraneous changes in this file (lines 3-5 and 22) will be reverted

Reply:
Yeah